plugins/jobs/src/js/types/Job.ts

```diff
@@ -0,0 +1,200 @@
+import {
+  JobDetailResponse as MetronomeJobDetailResponse,
+  JobStatus as MetronomeJobStatus
+} from "#SRC/js/events/MetronomeClient";
+import {
+  JobRunConnection,
+  JobRunConnectionTypeResolver,
+  JobRunConnectionSchema
+} from "#PLUGINS/jobs/src/js/types/JobRunConnection";
+import {
+  JobHistorySummary,
+  JobHistorySummaryTypeResolver,
+  JobHistorySummarySchema
+} from "#PLUGINS/jobs/src/js/types/JobHistorySummary";
+import {
+  JobRunStatusSummary,
+  JobRunStatusSummaryTypeResolver,
+  JobRunStatusSummarySchema
+} from "#PLUGINS/jobs/src/js/types/JobRunStatusSummary";
+import {
+  JobScheduleConnection,
+  JobScheduleConnectionTypeResolver,
+  JobScheduleConnectionSchema
+} from "#PLUGINS/jobs/src/js/types/JobScheduleConnection";
+import {
+  JobStatus,
+  JobStatusSchema
+} from "#PLUGINS/jobs/src/js/types/JobStatus";
+import { JobHistoryRun } from "#PLUGINS/jobs/src/js/types/JobHistoryRun";
+import {
+  JobDockerTypeResolver,
+  JobDocker,
+  JobDockerSchema
+} from "#PLUGINS/jobs/src/js/types/JobDocker";
+import { JobLabel, LabelSchema } from "#PLUGINS/jobs/src/js/types/JobLabel";
+import { cleanJobJSON } from "#SRC/js/utils/CleanJSONUtil";
+export interface Job {
```

Comment:
Could we add one empty line here to differentiate between import and export?

Reply:
Done